packages/sdk-client/test/auth.test/base-auth-flow.test.ts

```diff
@@ -414,22 +413,24 @@ describe('Base Auth Flow', () => {
 
     nock(middlewareOptions.host)
       .persist() // <-- use the same interceptor for all requests
-      .log(() => {
-        requestCount += 1
-      }) // keep track of the request count
       .filteringRequestBody(/.*/, '*')
-      .post('/oauth/token', '*')
+      .post('/oauth/token', () => {
+        requestCount += 1
+        return true
```

Comment:
Why do we have this here. I think you should remove this `true` so we can determine when the post errors out.

Reply:
This is to match anything in the body of `post`. It has the same meaning as `*`. I just make it a function and the function must return boolean if the body matches or not. It has nothing to do with errors.